webapp/src/components/call_widget/component.tsx

```diff
@@ -1236,7 +1285,94 @@ export default class CallWidget extends React.PureComponent<Props, State> {
                         }
                     </button>
                 </li>
-                {deviceType === 'output' && <li className='MenuGroup menu-divider'/>}
+            </React.Fragment>
+        );
+    };
+
+    renderChatThreadMenuItem = () => {
+        const {formatMessage} = this.props.intl;
+
+        // If we are on global widget we should show this
+        // only if we have the matching functionality available.
+        if (this.props.global && !window.desktopAPI?.openThread) {
+            return null;
+        }
+
+        return (
+            <React.Fragment>
```

Comment:
Just fyi, not important:
```suggestion
            <>
```